topydo/ui/Main.py

```diff
@@ -95,6 +95,8 @@ def __init__(self):
         self.todofile = TodoFile.TodoFile(config().todotxt())
         self.todolist = TodoList.TodoList(self.todofile.read())
 
+        self.pending_todos = []
```

Comment:
I did the renaming part, but I couldn't see the benefit of transition to `set()`. In current implementation we have to worry whether a todo item is already present in the marked list, as this is the best way to provide a toggle of the mark.